.travis.yml

```diff
@@ -11,6 +11,8 @@ before_install:
     - edm install -y --version 3.5 click setuptools
     - git clone git://github.com/force-h2020/force-bdss.git
     - pushd force-bdss && edm run -- python -m ci build-env && edm run -- python -m ci install && popd
+    - git clone --branch feature/custom_dataview git://github.com/force-h2020/force-wfmanager.git
+    - pushd force-wfmanager && edm run -- python -m ci install && popd
```

Comment:
I think the CI might be failing at this point, as the plugins now depend on the wfmanager.